Xamarin.Forms.Platform.Android/FastRenderers/AutomationPropertiesProvider.cs

```diff
@@ -55,6 +55,17 @@ internal static void SetAutomationId(AView control, VisualElement element, strin
 				defaultContentDescription = control.ContentDescription;
 
 			string value = ConcatenateNameAndHelpText(element);
+
+			var button = element as Button;
+			if (button != null)
+			{
+				var viewGroup = control as global::Android.Views.ViewGroup;
+				if (viewGroup != null)
+				{
+					control = viewGroup.GetChildAt(0);
```

Comment:
Can you explain why this is necessary? They are always equal 
![screenshot_4](https://user-images.githubusercontent.com/27482193/53355952-48dff000-393b-11e9-85d9-c50f71c9e817.png)


Also could you please use the pattern matching here
```C#
				if (element is Button && control is global::Android.Views.ViewGroup viewGroup)
				control = viewGroup.GetChildAt(0);
```

Reply:
I took the latter approach with `IButtonLayoutRenderer`.